orttraining/orttraining/test/python/orttraining_test_utils.py

```diff
@@ -46,30 +51,83 @@ def map_optimizer_attributes(name):
+                                                'device': {'id': device},
+                                                'mixed_precision': {
+                                                    'enabled': fp16,
+                                                    'loss_scaler': new_api_loss_scaler},
+                                                'debug': {'deterministic_compute': True, },
+                                                'utils': {
+                                                    'grad_norm_clip': True},
+                                                'distributed': {'allreduce_post_accumulation': True},
+                                                'lr_scheduler': new_api_lr_scheduler
+                                                })
+
+        param_optimizer = list(model.named_parameters())
+        params = [{
+            'params': [n for n, p in param_optimizer if "bias" in n or "LayerNorm.weight" in n],
+            "alpha": 0.9, "beta": 0.999, "lambda": 0.0, "epsilon": 1e-6}, {
+            'params': [n for n, p in param_optimizer if not ("bias" in n or "LayerNorm.weight" in n)],
+            "alpha": 0.9, "beta": 0.999, "lambda": 0.0, "epsilon": 1e-6}
+            ]
+
+        vocab_size = 99
+        new_model_desc = {
+            'inputs': [
+                ('input_ids', ['batch', 'max_seq_len_in_batch'],),
+                ('attention_mask', ['batch', 'max_seq_len_in_batch'],),
+                ('token_type_ids', ['batch', 'max_seq_len_in_batch'],),
+                ('masked_lm_labels', ['batch', 'max_seq_len_in_batch'],),
+                ('next_sentence_label', ['batch',])],
+            'outputs': [
+                ('loss', [1,], True),
+                ('prediction_scores', ['batch', 'max_seq_len_in_batch', vocab_size]),
+                ('seq_relationship_scores', ['batch', 2])]}
+
+        optim_config = optim.LambConfig(params=params, lr=2e-5)
+        model = orttrainer.ORTTrainer(model, new_model_desc, optim_config, options=options)
+    else:
+        model = ORTTrainer(model, None, model_desc, "LambOptimizer",
+            map_optimizer_attributes=map_optimizer_attributes,
+            learning_rate_description=IODescription('Learning_Rate', [1,], torch.float32),
+            device=device,
+            _enable_internal_postprocess=True,
+            gradient_accumulation_steps=gradient_accumulation_steps,
+            # BertLAMB default initial settings: b1=0.9, b2=0.999, e=1e-6
+            world_rank=args.local_rank, world_size=args.world_size,
+            use_mixed_precision=fp16,
+            allreduce_post_accumulation=allreduce_post_accumulation,
+            get_lr_this_step=get_lr_this_step if use_internal_get_lr_this_step else None,
+            loss_scaler=loss_scaler if use_internal_loss_scaler else None,
+            _opset_version=12)
 
     # trainig loop
     eval_batch = None
-    model.train()
+    if not use_new_api:
+        model.train()
```

Comment:
```suggestion
```
If we call `train_step` and `eval_step` manually as opposed to `model(input)`, do we still need to call `model.train()` and `model.eval()`